src/api/sfc-script-setup.md

```diff
@@ -302,7 +302,7 @@ export interface Props {
 
 const props = withDefaults(defineProps<Props>(), {
   msg: 'hello',
-  labels: () => ['one', 'two']
+  labels: ['one', 'two']
```

Comment:
The original text is correct. The default value for an array prop must be a factory function. You'll notice an [ESLint error](https://eslint.vuejs.org/rules/require-valid-default-prop.html) about this as well.

Reply:
Appreciate for you replay